core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -464,6 +464,8 @@ class LoggingMessageFormatter extends MessageFormatter   {
   private val defaultWriter: DefaultMessageFormatter = new DefaultMessageFormatter
   val logger = Logger.getLogger(getClass().getName)
 
+  override def init(props: Properties): Unit = defaultWriter.init(props)
```

Comment:
This was a bug in the `ConsolerConsumer` which was causing the `--print-timestamp=true` opt to not be respected.